### PR TITLE
Made Bank Items' names in messages configurable and added message for crystal deposit

### PR DESCRIPTION
**Pull request by Faun471**
I think peaches has a reason why the bank item names in messages aren't changeable, so I can remove it if needed

### CODE REVIEW

File: src/main/java/com/iridium/iridiumskyblock/bank/ExperienceBankItem.java [hunk 1]
```diff
@@ -50,12 +50,12 @@ public double withdraw(Player player, Number amount) {
                 player.sendMessage(StringUtils.color(IridiumSkyblock.getInstance().getMessages().bankWithdrew
                         .replace("%prefix%", IridiumSkyblock.getInstance().getConfiguration().prefix))
                         .replace("%amount%", String.valueOf(experience))
-                        .replace("%type%", "Experience")
+                        .replace("%type%", getName())
```
Comment (MertUnverdi) [inline src/main/java/com/iridium/iridiumskyblock/bank/ExperienceBankItem.java:53]: make these capitalized, it will return `experience`

Comment (MertUnverdi) [reply src/main/java/com/iridium/iridiumskyblock/bank/ExperienceBankItem.java:53]: or just add another getter in BankItems :panda:

**Review by MertUnverdi — CHANGES_REQUESTED** [review]

File: src/main/java/com/iridium/iridiumskyblock/bank/MoneyBankItem.java [hunk 2]
```diff
@@ -25,7 +25,7 @@
      * @param item          The Item which represents this bank item in the {@link com.iridium.iridiumskyblock.gui.BankGUI}
      */
     public MoneyBankItem(double defaultAmount, Item item) {
-        super("money", defaultAmount, true, item);
+        super("money", "money", defaultAmount, true, item);
```
Comment (MertUnverdi) [inline src/main/java/com/iridium/iridiumskyblock/bank/MoneyBankItem.java:28]:
```suggestion
        super("money", "Money", defaultAmount, true, item);
```

File: src/main/java/com/iridium/iridiumskyblock/bank/ExperienceBankItem.java [hunk 3]
```diff
@@ -85,12 +85,12 @@ public double deposit(Player player, Number amount) {
...
                 );
             }else{
                 player.sendMessage(StringUtils.color(IridiumSkyblock.getInstance().getMessages().insufficientFundsToDeposit
                         .replace("%prefix%", IridiumSkyblock.getInstance().getConfiguration().prefix))
-                        .replace("%type%", "Experience")
+                        .replace("%type%", getName())
```
Comment (MertUnverdi) [inline src/main/java/com/iridium/iridiumskyblock/bank/ExperienceBankItem.java:93]:
```suggestion
                        .replace("%type%", getDisplayName())
```

File: src/main/java/com/iridium/iridiumskyblock/bank/ExperienceBankItem.java [hunk 4]
```diff
@@ -85,12 +85,12 @@ public double deposit(Player player, Number amount) {
                 player.sendMessage(StringUtils.color(IridiumSkyblock.getInstance().getMessages().bankDeposited
                         .replace("%prefix%", IridiumSkyblock.getInstance().getConfiguration().prefix))
                         .replace("%amount%", String.valueOf(experience))
-                        .replace("%type%", "Experience")
+                        .replace("%type%", getName())
```
Comment (MertUnverdi) [inline src/main/java/com/iridium/iridiumskyblock/bank/ExperienceBankItem.java:88]:
```suggestion
                        .replace("%type%", getDisplayName())
```

File: src/main/java/com/iridium/iridiumskyblock/bank/ExperienceBankItem.java [hunk 5]
```diff
@@ -50,12 +50,12 @@ public double withdraw(Player player, Number amount) {
...
                 );
             }else{
                 player.sendMessage(StringUtils.color(IridiumSkyblock.getInstance().getMessages().insufficientFundsToWithdrew
                         .replace("%prefix%", IridiumSkyblock.getInstance().getConfiguration().prefix))
-                        .replace("%type%", "Experience")
+                        .replace("%type%", getName())
```
Comment (MertUnverdi) [inline src/main/java/com/iridium/iridiumskyblock/bank/ExperienceBankItem.java:58]:
```suggestion
                        .replace("%type%", getDisplayName())
```

File: src/main/java/com/iridium/iridiumskyblock/bank/ExperienceBankItem.java (hunk 1 above)
Comment (MertUnverdi) [inline src/main/java/com/iridium/iridiumskyblock/bank/ExperienceBankItem.java:53]:
```suggestion
                        .replace("%type%", getDisplayName())
```

File: src/main/java/com/iridium/iridiumskyblock/bank/ExperienceBankItem.java [hunk 6]
```diff
@@ -26,7 +26,7 @@
      * @param item          The Item which represents this bank item in the {@link com.iridium.iridiumskyblock.gui.BankGUI}
      */
     public ExperienceBankItem(double defaultAmount, Item item) {
-        super("experience", defaultAmount, true, item);
+        super("experience", "experience", defaultAmount, true, item);
```
Comment (MertUnverdi) [inline src/main/java/com/iridium/iridiumskyblock/bank/ExperienceBankItem.java:29]:
```suggestion
        super("experience", "Experience", defaultAmount, true, item);
```

File: src/main/java/com/iridium/iridiumskyblock/bank/CrystalsBankItem.java [hunk 7]
```diff
@@ -51,12 +51,12 @@ public double withdraw(Player player, Number amount) {
...
                         .replace("%prefix%", IridiumSkyblock.getInstance().getConfiguration().prefix))
                         .replace("%amount%", String.valueOf(crystals))
-                        .replace("%type%", "Crystals")
+                        .replace("%type%", getName())
                 );
             } else {
                 player.sendMessage(StringUtils.color(IridiumSkyblock.getInstance().getMessages().insufficientFundsToWithdrew
                         .replace("%prefix%", IridiumSkyblock.getInstance().getConfiguration().prefix))
-                        .replace("%type%", "Crystals")
+                        .replace("%type%", getName())
```
Comment (MertUnverdi) [inline src/main/java/com/iridium/iridiumskyblock/bank/CrystalsBankItem.java:59]:
```suggestion
                        .replace("%type%", getDisplayName())
```

File: src/main/java/com/iridium/iridiumskyblock/bank/CrystalsBankItem.java [hunk 8]
```diff
@@ -51,12 +51,12 @@ public double withdraw(Player player, Number amount) {
                 player.sendMessage(StringUtils.color(IridiumSkyblock.getInstance().getMessages().bankWithdrew
                         .replace("%prefix%", IridiumSkyblock.getInstance().getConfiguration().prefix))
                         .replace("%amount%", String.valueOf(crystals))
-                        .replace("%type%", "Crystals")
+                        .replace("%type%", getName())
```
Comment (MertUnverdi) [inline src/main/java/com/iridium/iridiumskyblock/bank/CrystalsBankItem.java:54]:
```suggestion
                        .replace("%type%", getDisplayName())
```

File: src/main/java/com/iridium/iridiumskyblock/bank/CrystalsBankItem.java [hunk 9]
```diff
@@ -25,7 +25,7 @@
      * @param item          The Item which represents this bank item in the {@link com.iridium.iridiumskyblock.gui.BankGUI}
      */
     public CrystalsBankItem(double defaultAmount, Item item) {
-        super("crystals", defaultAmount, true, item);
+        super("crystals", "crystals", defaultAmount, true, item);
```
Comment (MertUnverdi) [inline src/main/java/com/iridium/iridiumskyblock/bank/CrystalsBankItem.java:28]:
```suggestion
        super("crystals", "Crystals", defaultAmount, true, item);
```

**Review by MertUnverdi — CHANGES_REQUESTED** [review]
Also displayName will be null on exiting configs use an hardcode for setting defaults (i prefer Reflection#getCaller)

**Review by dlsf — APPROVED** [review]

**Pull request merged**